kafka-rest/pom.xml

```diff
@@ -156,6 +156,12 @@
             <artifactId>hamcrest-all</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.connectors</groupId>
+            <artifactId>jersey-apache-connector</artifactId>
+            <version>2.34</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
That's needed for the testing of the `PATCH` method, right? Can we skip the version if we're inheriting it from the parent POM? Feel free to ignore as this is being fast-tracked, so I don't want to request unwanted changes that aren't too important.